beacon-chain/blockchain/chain_info.go

```diff
@@ -525,3 +525,8 @@ func (s *Service) recoverStateSummary(ctx context.Context, blockRoot [32]byte) (
 	}
 	return nil, errBlockDoesNotExist
 }
+
+// BlockBeingSynced returns the HTR of the block being currently synced
```

Comment:
The comment is incorrect, we return a bool

Reply:
thanks, this is from a previous iteration, nice catch